Allclean

```diff
@@ -4,15 +4,15 @@
 echo "Cleaning logs and output files from previous runs..."
 
 rm -f precice-FLUID-events.json \
-      precice-STRUCTURE-events.json \
+      precice-Solid-events.json \
       precice-FLUID-iterations.log \
-      precice-STRUCTURE-convergence.log \
-      precice-STRUCTURE-iterations.log \
+      precice-Solid-convergence.log \
+      precice-Solid-iterations.log \
       precice-FLUID-events-summary.log \
       precice-STRUCTUR-events-summary.log \
       Postproc/*.vtk \
       Fluid.log \
-      Structure.log
+      Solid.log
```

Comment:
I understand that these files are not there anymore, right? Or are they from the Python side?

Reply:
that's true, I have removed them now:)